Taskfile.yml

```diff
@@ -30,7 +30,6 @@ tasks:
       - gofumpt -w .
       - go mod tidy
       - golangci-lint run --fix
-      - nilaway -fix -test=false -include-pkgs='github.com/opslevel/opslevel-go' ./...
```

Comment:
Removing this step - nilaway still has false positives and I've never gotten the `-fix` to actually work since it's still an experimental linter. 

If something is ever triggered by nilaway, we should fix it manually since it's likely a larger issue.